### PR TITLE
fix: the TRANSIENT_LOCAL topics were not working on server

### PR DESCRIPTION
**Pull request by Kapim**

### CODE REVIEW

File: era_5g_relay_network_application/data/service.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from multiprocessing import Queue
+from typing import Optional
+
+import rclpy
```
Comment (ZdenekM) [inline era_5g_relay_network_application/data/service.py]: Please add `# pants: no-infer-dep`.

File: era_5g_relay_network_application/data/topic.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import os
+from multiprocessing import Queue
+from typing import Any, Optional, Union
+
+import rclpy
```
Comment (ZdenekM) [inline era_5g_relay_network_application/data/topic.py]: Please add # pants: no-infer-dep.

**Review by ZdenekM — CHANGES_REQUESTED** [review]

File: era_5g_relay_network_application/data/service.py (hunk 1 above)
Comment (Kapim) [reply era_5g_relay_network_application/data/service.py]: added

File: era_5g_relay_network_application/data/topic.py (hunk 2 above)
Comment (Kapim) [reply era_5g_relay_network_application/data/topic.py]: added

File: era_5g_relay_network_application/worker_commands.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        self.command_queue = command_queue
+        self.outgoing_topics = outgoing_topics
+
+    def stop(self) -> None:
+        self.stop_event.set()
+
+    def run(self) -> None:
+        logging.debug(f"{self.name} thread is running.")
+
+        while True:
```
Comment (ZdenekM) [inline era_5g_relay_network_application/worker_commands.py]: Shouldn't this be `while not self.stop_event.is_set()`?

Comment (Kapim) [reply era_5g_relay_network_application/worker_commands.py]: yes, it was initially; I changed it during the tests and forgot to put it back... fixed

**Review by ZdenekM — APPROVED** [review]

**Pull request merged**